src/styles/utilities/_responsive.css

```diff
@@ -1,12 +1,12 @@
 .u-hidden-desktop {
   @media (--desktop) {
-    display: none;
+    display: none !important;
```

Comment:
again, here the usage of !important is completely legit and serves it's intended purpose.